[CI Failure] ND failure All post-commit tests - test_tosa_gather.py::test_tosa_gather_general

### Pipeline Name

All post-commit tests

### Test Name

test_tosa_gather.py::test_tosa_gather_general

### Path to Test in Repo

tests/ttnn/unit_tests/operations/test_tosa_gather.py

### Non-Deterministic (ND) failure?

Yes

### Failure type

Other

### Link to Failing Pipeline for non-ND or Custom Dispatch Run for ND

https://github.com/tenstorrent/tt-metal/actions/runs/16968433166/job/48100936516

### Failure Signature Snippet

```shell
FAILED tests/ttnn/unit_tests/operations/test_tosa_gather.py::test_tosa_gather_general[N=128-K=128-C=128-W=64] - AssertionError: 0.9997557511082162
```

### Additional info

```shell
https://github.com/tenstorrent/tt-metal/actions/runs/16968433166/job/48100936516#step:6:12481
https://github.com/tenstorrent/tt-metal/actions/runs/16968433166/job/48100936516

multiple instances
```

--- a/.github/workflows/ttsim.yaml
+++ b/.github/workflows/ttsim.yaml
@@ -282,8 +282,7 @@ jobs:
             tests/ttnn/unit_tests/operations/transformers/test_paged_cache_mask.py \
             tests/ttnn/unit_tests/operations/eltwise/test_polyval.py \
             tests/ttnn/unit_tests/operations/transformers/test_prefetcher_TG.py \
-            tests/ttnn/unit_tests/operations/data_movement/test_reallocate.py \
-            tests/ttnn/unit_tests/operations/data_movement/test_tosa_gather.py
+            tests/ttnn/unit_tests/operations/data_movement/test_reallocate.py
   ttsim-cpp-unit-tests:
     if: ${{ inputs.run-metal-cpp-unit-tests }}
     runs-on: tt-ubuntu-2204-small-stable

--- a/tests/ttnn/unit_tests/operations/data_movement/test_tosa_gather.py
+++ b/tests/ttnn/unit_tests/operations/data_movement/test_tosa_gather.py
@@ -8,7 +8,6 @@ import ttnn
 from tests.ttnn.utils_for_testing import assert_with_pcc
 
 
-@pytest.mark.skip("#26985: ND PCC failure, seems slight but APC must be stable")
 @pytest.mark.parametrize(
     "N, K, C, W",
     [

